data/refresh_materialized_views.sql

```diff
@@ -10,7 +10,7 @@ RETURNS INT AS $$
     FOR view IN SELECT matviewname FROM pg_matviews WHERE schemaname = schema_arg
     LOOP
       RAISE NOTICE 'Refreshing %.%', schema_arg, view.matviewname;
-      EXECUTE 'REFRESH MATERIALIZED VIEW CONCURRENTLY ' || schema_arg || '.' || view.matviewname;
```

Comment:
Won't dropping concurrently cause downtime on refreshes?

Reply:
Thanks for catching it. The change was inadvertent - I was trying to populate the materialized views for the 1st time and `CONCURRENTLY` could not be used in that case.